torch/distributed/_sharded_tensor/__init__.py

```diff
@@ -0,0 +1,41 @@
+from torch.distributed._sharding_spec import ShardingSpec
+from .api import (
+    ShardedTensor,
+)
+
+def empty(
```

Comment:
Do we need to consider `layout=torch.strided` here?

Reply:
I feel `process_grop` should probably be in the end since in most cases I expect you wouldn't specify it.